LayoutTests/imported/w3c/web-platform-tests/service-workers/service-worker/update-import-scripts.https.html

```diff
@@ -123,5 +123,13 @@
   assert_active_only(registration, expected_url);
 }, 'update() should find an update in an imported script but update() should ' +
    'result in failure due to missing the other imported script.');
+
+promise_test(async t => {
+    const [registration, expected_url] = await prepare_ready_normal_worker(
+    t, 'import-scripts-cross-origin-worker.sub.js');
```

Comment:
This seems like it should be indented, or on the previous line

Reply:
Changed the indention of the whole test to match the rest of the file.